src/public/authentication.ts

```diff
@@ -138,7 +138,7 @@ export namespace authentication {
     height = Math.min(height, Communication.currentWindow.outerHeight - 200);
     // Convert any relative URLs into absolute URLs before sending them over to the parent window
     const link = document.createElement('a');
-    link.href = authParams.url;
+    link.href = authParams.url.replace('{oauthRedirectMethod}', 'web');
```

Comment:
Should we just do this when `isExternal` is set to true?

Reply:
has no effect when isExternal = false